libs/events/src/types/events.ts

```diff
@@ -24,15 +24,15 @@ export enum CowEvents {
 }
 
 // Define types for event payloads
-export interface CowEventPayloadMap {
-  [CowEvents.ON_TOAST_MESSAGE]: OnToastMessagePayload
-  [CowEvents.ON_POSTED_ORDER]: OnPostedOrderPayload
-  [CowEvents.ON_FULFILLED_ORDER]: OnFulfilledOrderPayload
-  [CowEvents.ON_CANCELLED_ORDER]: OnCancelledOrderPayload
-  [CowEvents.ON_EXPIRED_ORDER]: OnExpiredOrderPayload
-  [CowEvents.ON_PRESIGNED_ORDER]: OnPresignedOrderPayload
-  [CowEvents.ON_ONCHAIN_TRANSACTION]: OnTransactionPayload
-  [CowEvents.ON_CHANGE_TRADE_PARAMS]: OnTradeParamsPayload
+export interface CowWidgetEventPayloadMap {
```

Comment:
This is a breaking change for downstream consumers, right?

Reply:
Actually two:
```
CowEventPayloadMap -> CowWidgetEventPayloadMap
CowEvents -> CowWidgetEvents
```